.github/workflows/ci.yml

```diff
@@ -15,3 +15,6 @@ jobs:
   call-terraform-ci-pipeline:
     uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-terraform-module-ci.yml@v1.7.3
     secrets: inherit
+    with:
+      craTarget: "patterns/roks"
+      craGoalIgnoreFile: "cra-tf-validate-ignore-goals.json"
```

Comment:
please remove the default values you added in patterns/roks/variables.tf and instead pass them as TF_VARs here:

`craEnvironmentVariables: "TF_VAR_prefix=land-zone-roks,TF_VAR_region=us-south"`

Reply:
Done